index.html

```diff
@@ -137,13 +177,66 @@ <h2>Professional Art<br>Printing Data</h2>
             <button class="seeProject-btn color-primary button">See Project</button>
         </div>
     </div>
+    <div class="work-grids-desktop color-primary">
+        <div class="work-grid-1 work-grid-desktop">
+            <button class="seeProject-btn color-primary button">See project</button>
+        </div>
```

Comment:
For the cards in the works section, there are two states of the cards. And we need the "See Project" button for each card for sure. Kindly check the image below to understand the default and hover state of the cards and fix your cards accordingly. 👍
<img width="647" alt="Screenshot 2023-01-20 at 8 53 55 AM" src="https://user-images.githubusercontent.com/25887253/213627406-7c8bf13e-a838-4ede-a3c0-eccd3e1b2f2c.png">

Reply:
@huseyinbiyik hello sir, could you please help me with this part? I've been stuck for hours on this problem.

Reply:
Hello @matee-safi, unfortunately, I am not available right now. But you can check this [link](https://github.com/Yosefgeda/My-Portfolio/pull/5) to get inspired.